api/envoy/extensions/filters/http/jwt_authn/v3/config.proto

```diff
@@ -717,3 +721,14 @@ message PerRouteConfig {
     string requirement_name = 2 [(validate.rules).string = {min_len: 1}];
   }
 }
+
+// This message specifies a combination of header name and claim name.
+message JwtClaimToHeader {
+  
+  // The HTTP header name.
```

Comment:
Same comments as in the previous PR, LMK when this is ready for review.

Reply:
PTAL

Reply:
To answer you previous comment - This has been a popular ask, so it's good to add a standalone field for this. And even if we use dynamic metadata, we need the claim name and header name from user in some form. IMO this could be the easiest solution from user point of view.